phylogenetic/build-configs/inrb/curate_private_data.py

```diff
@@ -0,0 +1,265 @@
+from typing import Any
+from Bio import SeqIO
+import argparse
+from openpyxl import load_workbook
+from datetime import datetime
+from os import path, mkdir
+from sys import exit
+
+Sequences = dict[str, SeqIO.SeqRecord]
+Metadata = dict[str, dict[str, str]]
+MetadataHeader = list[str]
+
+# The following seem reasonable to hardcode, as they're central to the current mpox workflows
+DATE_COLUMN = 'date'
+ACCESSION = 'accession'
```

Comment:
for consistency, consider: 
```suggestion
ACCESSION_COLUMN = 'accession'
```

Reply:
[Done](https://github.com/nextstrain/mpox/compare/687499532247b1b2f5064055830771ed179addeb..42364fdb31e4b7239fb1b491ade4b61337856184) and also squashed the fixup commit.